<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>776</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_calc">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>649</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Calculator</string>
     </attribute>
     <widget class="QPushButton" name="b_acos">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>400</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>acos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_sqrt">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>460</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_cos">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_value">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-9999999999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999999999.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="b3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>500</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_log">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>460</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b0">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_tan">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_tilde">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>280</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_enter">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_plus">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>220</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_usingx">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>using x=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_skobk_l">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>51</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>490</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_x">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>110</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_expression">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="b_mult">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>220</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_skobk_r">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>51</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_ln">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_divide">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_sin">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_minus">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>220</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_pow">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>370</y>
        <width>111</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_asin">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>asin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_atan">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>400</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>atan</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_graph">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>649</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Graph</string>
     </attribute>
     <widget class="QPushButton" name="b_create_graph">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>520</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Create graph</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>571</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_end_value">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_specifyendx">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>550</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>specify end x:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_specifyendx_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>510</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>specify x range in plot</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_range_start">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>530</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>-4.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_range_end">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>530</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>4.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>530</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>from</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>530</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="y_range_start">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>580</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>-3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>580</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="y_range_end">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>580</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-1000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>580</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>from</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_specifyendx_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>560</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>specify y range in plot</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_specifyendx_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>600</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>specify step</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="step_value">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>620</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_specifyendx_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>specify start x:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_start_value">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-9999.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9999.000000000000000</double>
      </property>
      <property name="value">
       <double>-3.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_credit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>649</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Credit calculator</string>
     </attribute>
     <widget class="QCheckBox" name="cb_is_differentiated">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>is differentiated</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="credit_amount">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="months_to_pay">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="interest_rate">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.001000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="b_credit_calculate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Credit amount</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Months to pay</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interest rate (per year)</string>
      </property>
     </widget>
     <widget class="QLabel" name="hidden_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Monthly payment</string>
      </property>
     </widget>
     <widget class="QLabel" name="hidden_1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total payment</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="monthly_payment">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>320</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="hidden_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Overpayment</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="total_payment">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>420</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="overpayment">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
